examples/acoustic/acoustic_example.py

```diff
@@ -24,7 +24,7 @@ def source(t, f0):
     return (1-2.*r**2)*np.exp(-r**2)
 
 
-def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0), tn=250.0,
+def run(dimensions=(150, 150, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
```

Comment:
That's why it needs to be changed in `benchmark.py` and `tti_example.py` as well